test/common/http/http2/codec_impl_test.cc

```diff
@@ -47,11 +47,12 @@ class Http2CodecImplTestFixture {
 public:
   struct ConnectionWrapper {
     void dispatch(const Buffer::Instance& data, ConnectionImpl& connection) {
+      Http::Status status;
       buffer_.add(data);
       if (!dispatching_) {
         while (buffer_.length() > 0) {
           dispatching_ = true;
-          connection.dispatch(buffer_);
+          status = connection.innerDispatch(buffer_);
```

Comment:
I felt bad about writing this line. FYI.

Reply:
Can you maybe add a comment/TODO to reference exception removal for someone that comes across this?

Reply:
done